superset/extensions.py

```diff
@@ -0,0 +1,45 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Yes!

Reply:
I think I tried to do this at least twice in the past and got tangled up, never was able to push through. Let's get this through this time.